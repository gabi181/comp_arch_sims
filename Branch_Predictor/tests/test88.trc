8 2 0 1 local_history local_tables not_using_share
0x78CAFCE8 N 0x2FBA6C18
0x78CAFCE8 N 0x2FBA6C18
0x78CAFCE8 T 0x2FBA6C18
0x78CAFCE8 N 0x2FBA6C18
0x78CAFCE8 T 0x2FBA6C18
0x78CAFCE8 T 0x2FBA6C18
0x78CAFCE8 N 0x2FBA6C18
0x78CAFCE8 T 0x2FBA6C18
0x78CAFCE8 N 0x2FBA6C18
0x78CAFCE8 N 0x2FBA6C18
0x78CAFCE8 N 0x2FBA6C18
0x78CAFCE8 N 0x2FBA6C18
0x78CAFCE8 N 0x2FBA6C18
0x78CAFCE8 N 0x2FBA6C18
0x78CAFCE8 N 0x2FBA6C18
0x78CAFCE8 N 0x2FBA6C18
0x78CAFCE8 N 0x2FBA6C18
0x78CAFCE8 T 0x2FBA6C18
