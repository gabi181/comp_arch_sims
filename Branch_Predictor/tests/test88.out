0x78cafce8 N 0x78cafcec
0x78cafce8 N 0x78cafcec
0x78cafce8 N 0x78cafcec
0x78cafce8 N 0x78cafcec
0x78cafce8 N 0x78cafcec
0x78cafce8 N 0x78cafcec
0x78cafce8 N 0x78cafcec
0x78cafce8 T 0x2fba6c18
0x78cafce8 N 0x78cafcec
0x78cafce8 T 0x2fba6c18
0x78cafce8 N 0x78cafcec
0x78cafce8 N 0x78cafcec
0x78cafce8 N 0x78cafcec
0x78cafce8 N 0x78cafcec
0x78cafce8 N 0x78cafcec
0x78cafce8 N 0x78cafcec
0x78cafce8 N 0x78cafcec
0x78cafce8 N 0x78cafcec
flush_num: 5, br_num: 18, size: 328b
